api/users/views.py

```diff
@@ -495,19 +511,22 @@ class UserNodes(JSONAPIBaseView, generics.ListAPIView, UserMixin, NodeODMFilterM
     view_category = 'users'
     view_name = 'user-nodes'
 
-    ordering = ('-date_modified',)
-
-    # overrides ODMFilterMixin
-    def get_default_odm_query(self):
+    # overrides NodesFilterMixin
+    def get_default_queryset(self):
         user = self.get_user()
         query = MQ('contributors', 'eq', user) & default_node_list_query()
         if user != self.request.user:
             query &= default_node_permission_query(self.request.user)
-        return query
+        return Node.find(query)
 
     # overrides ListAPIView
     def get_queryset(self):
-        return Node.find(self.get_query_from_request()).select_related('node_license').include('guids', 'contributor__user__guids', 'root__guids', limit_includes=10)
+        return (
+            Node.objects.filter(id__in=set(self.get_queryset_from_request().values_list('id', flat=True)))
```

Comment:
Why the extra query here? Can you just chain calls to `self.get_queryset_from_request()`?

```python
self.get_queryset_from_request()
.select_related(...)
...
```